### DISCUSSION TITLE
RFC 2725 compliance.

### DISCUSSION THREAD

**Discussion by dhutchin-ctl** [irrdnet/irrd#744] [Q&A]
Under the heading of "Object validation in IRRd" (version 4.2.6 at least), it says:
     In` general, IRRd follows RFC 2622, 2650, 2726, 2725, 4012 and 2769. 
However the functionality of section 9.6 of the RFC does not appear to be allowed unless the validation mode in IRRd is set to "non-strict".  Does this seems reasonable?
I would think that the elements of a set's "members:" attribute should allow any element to be prefixed with an explicit reference to another registry.  How that reference is used by the clients of the registry is a totally different can of worms.  I like leaving the validation mode as "strict", but I would like to have cross registry references.

**Comment by mxsasha** [comment]
Oh, interesting, I had not seen this before. Something like this was [brought up in RIPE routing-wg](https://www.ripe.net/ripe/mail/archives/routing-wg/2022-November/004649.html) recently but no conclusions were reached. The :: notation is used informally here and there, but does not have wide adoption. There is potential there, but as raised in that thread, backwards compatibility is an issue. 

I do think this makes sense and adds real value for (as-)set referrals, but IRRd indeed does not allow it for person/role. I don't see a lot to be gained there. There is no support anywhere I know of that implements the referral maintainer scheme, and that would be much more complex, so I see it as a separate thing. It would also require the major operators to want this feature.
